### ISSUE TITLE
Getting undefined when open a modal

### ISSUE THREAD

**Issue by utiq** [valor-software/ngx-bootstrap#2937]
I'm opening a modal in this way:

```
let itemId = '12345';
this.bsModalRef = this.modalService.show(MyComponent);
this.bsModalRef.content.itemId = itemId;
```
In my.component.html I can see itemId is correctly working:
{{ itemId }} (I see 12345 in chrome)

But when I want to check itemId in my.component.ts
```
ngOnInit() {
  console.log(itemId)
}
```
itemId is undefined

**Comment by davidrdz93** [comment]
Same problem! Try to leave a breakpoint where you call  show method and you willl see that line where you assing itemId is jumped! ngOnInit is being call right after show method..

**Comment by IlyaSurmay** [comment]
Take a look at this - https://github.com/valor-software/ngx-bootstrap/issues/2354#issuecomment-320944042
There's a fix #2600 that isn't merged yet.
